umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif9/spatial_extent.clj

```diff
@@ -0,0 +1,127 @@
+(ns cmr.umm-spec.xml-to-umm-mappings.dif9.spatial-extent
+  "Defines mappings from DIF 9 Spatial_Coverage elements into UMM records"
+  (:require
+   [clojure.string :as string]
+   [cmr.common.xml.parse :refer :all]
+   [cmr.common.xml.simple-xpath :refer [select text]]))
+
+;; Bounding Box Parsing
+;; This is the DIF 9 bounding box parsing map
+(def bounding-box-map {:NorthBoundingCoordinate "Northernmost_Latitude"
+                        :SouthBoundingCoordinate "Southernmost_Latitude"
+                        :WestBoundingCoordinate "Westernmost_Longitude"
+                        :EastBoundingCoordinate "Easternmost_Longitude"})
+
+(defn- build-bounding-rectangle
+  "This function uses recursion to peice together all of the horizontal domain coordinates into
+   a map without nil values. The return value is a map of either the coordinates or nil.
+   {:WestBoundingCoordinate \"-180\",
+    :NorthBoundingCoordinate \"-90\",
+    :EastBoundingCoordinate \"180\",
+    :SouthBoundingCoordinate \"90\"}
+   Nil values will not exist in the map."
+  [spatial-coverage bounding-box-map result]
```

Comment:
I think we should rename bounding-box-map since it's the same as the def, but then you're doing the `(drop 1)` on it. Made it really confusing to read.